reporters_db/data/reporters.json

```diff
@@ -1597,10 +1595,7 @@
             },
             "mlz_jurisdiction": [],
             "name": "Neutral citation for Court of International Trade",
-            "variations": {},
```

Comment:
I'm confused. Most of the other entries don't need empty fields?

Reply:
I did not add this variations empty value.  But it was not breaking any tests so I did not remove it.